### DISCUSSION TITLE
Media URI

### DISCUSSION THREAD

**Discussion by Stackjaeger** [HugoFara/lwt#130] [Q&A] [answered]
I install it via docker but I can't find the directory "../html/media" to add audio
![Screenshot_1](https://github.com/HugoFara/lwt/assets/43479365/3abd7ec2-e295-41d3-942f-8689c06da7fd)

**Comment by HugoFara** [comment, marked as answer]
Hello! Did you use [lwt-docker-installer](https://github.com/HugoFara/lwt-docker-installer) or did you build the container from source?

If you used the Docker installer, I just published a fix, you can again now!

If you built from source, I did not encounter the same issue, but you can try to replace ``./media:/var/www/media`` by ``./media:/var/www/html/media`` at line 18 of ``docker-compose.yml``. I think this is the source of your troubles.

Tell me if it works!

**Reply by Stackjaeger** [reply]
yes, it worked thanks a lot :D
